tests/golden/copy-command/copy-command.bats

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bats
+
+load '../helpers/bats-support/load'
+load '../helpers/bats-assert/load'
+load '../helpers'
+
+@test "copy an entry" {
+  run coffer create /a
+  assert_output "[SUCCESS] Entry created at '/a'."
+
+  run coffer copy /a /b
+  assert_output "[SUCCESS] Copied '/a' to '/b'."
+
+  run cleanOutput coffer view /
+  cat copy-an-entry.gold | assert_output -
+}
+
+@test "copy a directory" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer copy /a /
+  assert_output "[SUCCESS] Copied '/a/b/c' to '/b/c'."
+
+  run cleanOutput coffer view /
+  cat copy-a-directory.gold | assert_output -
+}
+
+@test "copy one directory to other" {
+  run coffer create /a/b/c
+  assert_output "[SUCCESS] Entry created at '/a/b/c'."
+
+  run coffer create /c/d/e
+  assert_output "[SUCCESS] Entry created at '/c/d/e'."
+
+  run coffer copy /a /c
+  assert_output "[SUCCESS] Copied '/a/b/c' to '/c/b/c'."
+
+  run cleanOutput coffer view /
+  cat copy-one-directory-to-other.gold | assert_output -
+}
+
+@test "trying to copy an entry to directory" {
```

Comment:
We should add a similar test that checks that an entry cannot overwrite a directory **even if** `--force` is used.

Reply:
Done